source/standards/disaster-recovery.html.md.erb

```diff
@@ -0,0 +1,77 @@
+
+- Domain Name System (DNS) provider
+- Content Delivery Network (CDN)
+- any Virtual Private Networks (VPNs) your development team might use to access production systems
+build pipelines
+- external library and container registries (for example, RubyGems, PyPI or Docker Hub)
+- secrets management providers
+
+### Back up your data
+
+You should back up your data. How often you back up will depend on your [Recovery Point Objective (RPO)](https://cloud.google.com/architecture/dr-scenarios-planning-guide#basics_of_dr_planning). An RPO determines what you consider an acceptable loss of data between the last backup and the point of data loss. For example, if you back up your data nightly, your RPO is 24 hours, the length of time between the backups. In practice, many modern database service cloud providers provide [point-in-time recovery](https://docs.aws.amazon.com/aws-backup/latest/devguide/point-in-time-recovery.html) from backups. This can result in an RPO of around 5 minutes or less.
+
+### Have cold or offline backups
+
+A cold or offline backup is a copy of your backup, isolated from the rest of your production system. Offline backups help protect you from some disaster scenarios. If an attacker gains access to your live database service and your backups, you could experience a ransomware attack, or risk having your entire data estate deleted. You should implement offline backups so that even users with administrative access cannot alter or delete data. 
+
+Offline backups also help protect your data from accidental deletion. For example, copy your backups to a different account with limited access rights. Make sure:
+
+- the backups are encrypted and versioned
+- the account you are copying the backups from only has write permissions
+
+### Decide where to store your backups
+
+Where you are storing your backups is also important for disaster recovery. Most cloud storage solutions have durability guarantees. However, an outage could occur in the region containing your live service and backups. If your service has very high availability requirements, you may need copies of your data in a different region, or with a different service provider. Your conversations with IA and service owners can help you determine your risk tolerance for these hopefully rare scenarios.
+
+The National Cyber Security Centre (NCSC) article [Offline backups in an online world](https://www.ncsc.gov.uk/blog-post/offline-backups-in-an-online-world) further explains why offline backups are important and suggests some approaches to use in a cloud environment.
+
+### Decide on manual or automatic recovery in the event of a disaster
```

Comment:
This section feels like it's bundling a few different things together:

* Setting RTOs to drive decisions about how you design your recovery mechanisms
* The fact that different elements of a service (or the same elements under different conditions) might require different RTOs
* Different uses of manual vs. automatic recovery, and the fact that these also have different cost and complexity trade-offs
* That risk and service owners are the ones who ultimately own (set) the RTOs and RPOs

Possibly because of this mixture, the current section title feels very reductive, and not particularly descriptive of what you're trying to convey.

Reply:
Yep you're right. I'll change it